scripts/prepare_for_tests.sh

```diff
@@ -4,7 +4,7 @@ set -e
 COMPILER_DIRECTORY="$(git rev-parse --show-toplevel)/crates/cast/tests/utils/compiler"
 CAIRO_REPO="https://github.com/starkware-libs/cairo/releases/download"
 
-SCARB_VERSIONS=("0.4.1" "0.5.2")
```

Comment:
I think we should leave 0.4.1 as it was. We want to check a "range" of scarb versions